valtimo-dependencies/build.gradle

```diff
@@ -53,18 +53,26 @@ dependencies {
         api("com.ritense.valtimo:form-link:${projectVersion}!!")
         api("com.ritense.valtimo:form:${projectVersion}!!")
         api("com.ritense.valtimo:keycloak-iam:${projectVersion}!!")
+        api("com.ritense.valtimo:local-document-generation:${projectVersion}!!")
+        api("com.ritense.valtimo:milestones:${projectVersion}")
         api("com.ritense.valtimo:notes:${projectVersion}!!")
         api("com.ritense.valtimo:plugin-valtimo:${projectVersion}!!")
         api("com.ritense.valtimo:plugin:${projectVersion}!!")
+        api("com.ritense.valtimo:portaaltaak:${projectVersion}!!")
         api("com.ritense.valtimo:process-document:${projectVersion}!!")
         api("com.ritense.valtimo:process-link:${projectVersion}!!")
+        api("com.ritense.valtimo:smartdocuments:${projectVersion}!!")
         api("com.ritense.valtimo:temporary-resource-storage:${projectVersion}!!")
+        api("com.ritense.valtimo:test-utils-common:${projectVersion}!!")
         api("com.ritense.valtimo:value-resolver:${projectVersion}!!")
         api("com.ritense.valtimo:web:${projectVersion}!!")
+        api("com.ritense.valtimo:wordpress-mail:${projectVersion}!!")
```

Comment:
`wordpress-mail` is a custom made plugin for Den Haag. It can't be used by other municipalities. I think it shouldn't be in this list

Reply:
This is just a version constraint, it will not force the dependency. So I think it's fine.